cli/command/system/info.go

```diff
@@ -261,7 +262,19 @@ func printSwarmInfo(dockerCli command.Cli, info types.Info) {
 		fmt.Fprintln(dockerCli.Out(), " ClusterID:", info.Swarm.Cluster.ID)
 		fmt.Fprintln(dockerCli.Out(), " Managers:", info.Swarm.Managers)
 		fmt.Fprintln(dockerCli.Out(), " Nodes:", info.Swarm.Nodes)
+		var strAddrPool strings.Builder
+		if info.Swarm.Cluster.DefaultAddrPool != nil {
+			for _, p := range info.Swarm.Cluster.DefaultAddrPool {
+				strAddrPool.WriteString(p + "  ")
+			}
+			fmt.Fprintln(dockerCli.Out(), " Default Address Pool:", strAddrPool.String())
+			fmt.Fprintln(dockerCli.Out(), " SubnetSize:", info.Swarm.Cluster.SubnetSize)
+		} else {
+			fmt.Fprintln(dockerCli.Out(), " Default Address Pool:", "10.0.0.0/8")
```

Comment:
Just noticed this, but wondering if this is correct; Is this the only pool that swarmkit uses?

Also; if this default is known daemon-side, it should be returned by the daemon (i.e., we should not hard-code the value here)

Reply:
Yes. This is the only pool we have hardcoded in libnetwork. As this information is not available in swarmkit we dont store it in cluster object currently. I can set it in moby instead of CLI here. 
Either I can remove when I create new moby PR and update it or I can remove it here now and later update moby code with new PR. let me know how do  you prefer. I can fix it.

Reply:
It's better to remove it here, and only print the information we actually get from the API.

> This is the only pool we have hardcoded in libnetwork. As this information is not available in swarmkit we dont store it in cluster object currently.

Perhaps it's worth considering to define a default either in moby, or in swarmkit, so that the pool is always initialised (also from a swarmkit perspective); we can discuss, and change that after this PR is merged though

Reply:
I initially thought of displaying only if the user configure it. But later I thought people will complain if they dont see output in in built case.  will remove from here and update it on Moby